diff -ruN glproto-1.4.5./glxproto.h glproto-1.4.5/glxproto.h
--- glproto-1.4.5./glxproto.h	2006-03-11 23:16:28.000000000 +0100
+++ glproto-1.4.5/glxproto.h	2006-03-11 23:29:08.983113088 +0100
@@ -2524,4 +2524,47 @@
 #define X_GLvop_CheckFramebufferStatusEXT              1427
 #define X_GLvop_GetFramebufferAttachmentParameterivEXT 1428
 
+/* XXX. GL_ARB_vertex_program */
+#define X_GLvop_VertexAttrib1svARB			4189
+#define X_GLvop_VertexAttrib1fvARB			4193
+#define X_GLvop_VertexAttrib1dvARB			4197
+#define X_GLvop_VertexAttrib2svARB			4190
+#define X_GLvop_VertexAttrib2fvARB			4194
+#define X_GLvop_VertexAttrib2dvARB			4198
+#define X_GLvop_VertexAttrib3svARB			4191
+#define X_GLvop_VertexAttrib3fvARB			4195
+#define X_GLvop_VertexAttrib3dvARB			4199
+#define X_GLvop_VertexAttrib4bvARB			4230
+#define X_GLvop_VertexAttrib4svARB			4192
+#define X_GLvop_VertexAttrib4ivARB			4231
+#define X_GLvop_VertexAttrib4ubvARB			4232
+#define X_GLvop_VertexAttrib4usvARB			4233
+#define X_GLvop_VertexAttrib4uivARB			4234
+#define X_GLvop_VertexAttrib4fvARB			4196
+#define X_GLvop_VertexAttrib4dvARB			4200
+#define X_GLvop_VertexAttrib4NbvARB			4235
+#define X_GLvop_VertexAttrib4NsvARB			4236
+#define X_GLvop_VertexAttrib4NivARB			4237
+#define X_GLvop_VertexAttrib4NubvARB			4201
+#define X_GLvop_VertexAttrib4NusvARB			4238
+#define X_GLvop_VertexAttrib4NuivARB			4239
+#define X_GLvop_BindProgramARB				4180
+#define X_GLvop_ProgramEnvParameter4fvARB		4184
+#define X_GLvop_ProgramEnvParameter4dvARB		4185
+#define X_GLvop_ProgramLocalParameter4fvARB		4215
+#define X_GLvop_ProgramLocalParameter4dvARB		4216
+#define X_GLvop_ProgramStringARB			4217
+#define X_GLvop_DeleteProgramsARB			1294
+#define X_GLvop_GenProgramsARB				1295
+#define X_GLvop_GetProgramEnvParameterfvARB		1296
+#define X_GLvop_GetProgramEnvParameterdvARB		1297
+#define X_GLvop_GetProgramLocalParameterfvARB		1305
+#define X_GLvop_GetProgramLocalParameterdvARB		1306
+#define X_GLvop_GetProgramivARB				1307
+#define X_GLvop_GetProgramStringARB			1308
+#define X_GLvop_GetVertexAttribdvARB			1301
+#define X_GLvop_GetVertexAttribfvARB			1302
+#define X_GLvop_GetVertexAttribivARB			1303
+#define X_GLvop_IsProgramARB				1304
+
 #endif /* _GLX_glxproto_h_ */
